model/Core/Classes/PackageVerificationCode.md

```diff
@@ -18,18 +18,21 @@ This identifier enables a recipient to determine if any file in the original pac
 
 Algorithm:
```

Comment:
Why has this algorithm been changed without wider review? 
It's not clear to me that the new version is clearer - or even accurate.   Please revert.

Reply:
Well, let's see about the old one. Here are the issues:

- it starts by assigning `verificationcode = 0` and this is never used or mentioned before, since the final line computes `hashValue`
- it assigns `filelist` to an empty string and then reassigns it without using this empty string, so this assignment is useless
- it has templist as a string (not a list), appends to it values plus `/n` -- without defining this plus, and of course newlines are `\n`
- then it says "sort templist" without explaining how to sort a single string
- then it computes a `hashValue` by calculating `algorithm(filelist)` ... while having a comment that `algorithm()` reads a file and does not take content as argument.

I only went in to fix the very long comment line which did not fit in the page.

But reading the old pseudo-algorithm, I found out it makes no sense. @goneall , what do you think?